### ISSUE TITLE
Rendering time axis tick labels when zoomed in

### ISSUE THREAD

**Issue by bendichter** [flatironinstitute/neurosift#167]
https://neurosift.app/?p=/nwb&url=https://api.dandiarchive.org/api/assets/7d1990d7-a714-406e-9cb2-1b8f4015869e/download/&dandisetId=000971&dandisetVersion=draft&tab=neurodata-item:/stimulus/presentation/OptogeneticSeries|OptogeneticSeries&tab-time=1260.6075838867034,1281.6234892105049,1926.1276592483853

<img width="1395" alt="image" src="https://github.com/flatironinstitute/neurosift/assets/844306/11c43616-6f78-4b35-86dc-c474cda79650">

When zoomed far in on a TimeSeries, it's not clear what the starting time is. In this plot, you can see second tick labels from 1 to 21, but you can't tell that you are also 21 minutes into the recording until you zoom out:

https://github.com/flatironinstitute/neurosift/assets/844306/05de3337-c6a0-4cc5-904b-318404ba5dd9

Could we add a "21 min+" on the lower left to indicate what the start of this interval is?

**Comment by magland** [comment]
I agree there is a problem here, but I don't think it would be very easy for me to implement in a general way. Note that you do get the full start time on the left panel. I'll think about this some more... but not going to tackle it right now.

**Comment by bendichter** [comment]
oh, I see. OK, now that I see the start time is present on the left I don't feel this is that urgent
